website/src/pages/extensions/index.tsx

```
@@ -1,24 +1,682 @@
+                    <span className="font-medium text-charcoal-300 dark:text-white">{extension.latestVersion}</span>
+                  </div>
+                  <div className="flex justify-between">
+                    <span className="text-gray-600 dark:text-gray-300">Publisher</span>
+                    <span className="font-medium text-charcoal-300 dark:text-white">{extension.publisher}</span>
+                  </div>
+                  <div className="flex flex-col sm:flex-row sm:justify-between gap-2">
+                    <span className="text-gray-600 dark:text-gray-300">Categories</span>
+                    <div className="flex flex-wrap gap-1 sm:justify-end">
+                      {extension.categories.map((cat, index) => (
+                        <span
+                          key={index}
```

Biome (2.1.2)

[error] Avoid using the index of an array as key property in an element.

This is the source of the key value.

The order of the items may change, and this also affects performances and component state.
Check the React documentation. 

(lint/suspicious/noArrayIndexKey)